draft-ietf-cats-metric-definition.md

```diff
@@ -447,20 +464,6 @@ Therefore, from a stability standpoint, Level 1 and Level 2 metrics are preferre
 
 In conclusion, for CATS, Level 2 metrics are recommended due to their simplicity and minimal protocol overhead. If more advanced scheduling capabilities are required, Level 1 metrics offer a balanced approach with manageable complexity. While Level 0 metrics are the most detailed and dynamic, their high overhead makes them unsuitable for direct transmission to network devices and thus not recommended for standard protocol integration.
 
-# Implementation Guidance on Using CATS Metrics
```

Comment:
@bpatient78, just for the record, could you explain why this section is being removed from the document? thanks.